Deprecate full download via soap API (#635)
OpenEnergyPlatform/open-MaStR: docs/getting_started.md

API download
-----------------------------------
!!! warning "API download is deprecated"

>     Since `v0.16.0` the `download(method="API")` is deprecated and cannot be used anymore. The default behaviour changes `method="API"` to `method="bulk"`. You can still use the basic SOAP API functionalities from the [`MaStRAPI`][open_mastr.soap_api.download.MaStRAPI] class.

Reviewer: ```suggestion
!!! warning "API download has been removed"
    In versions > `v0.16.0` the `download(method="API")` cannot be used anymore. The default behaviour changes `method="API"` to `method="bulk"`. You can still use the basic SOAP API functionalities from the [`MaStRAPI`][open_mastr.soap_api.download.MaStRAPI] class.
```
